dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/jpa22/JPA22Injection.java

```diff
@@ -24,7 +33,17 @@
 
     @BeforeClass
     public static void setUp() throws Exception {
-        ShrinkHelper.defaultApp(server1, APP_NAME, "jpa22injection.web", "jpa22injection.entity");
+        final String resPath = "test-applications/jpa22/" + APP_NAME + "/resources/";
```

Comment:
do you really want to put the apps in a `test-applications/jpa22/` folder as opposed to just the `test-applications/` folder?  This will mean a lot of the default app packaging stuff will not work, and your `APP_NAME` already contains the string `jpa22`.

Reply:
I would like the extra level of organization in place, it would make it easier for me to maintain it with FATs for earlier JPA spec versions if I can compartmentalize the FATs per spec, etc.